src/main/resources/db/migration/0250/V272__ParticipantProjectSpecies.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE accelerator.participant_project_species (
+    id BIGINT PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
This is fine, but to point out an alternative: the primary key could be `(project_id, species_id)` here.

Reply:
Would that allow me to get rid of the unique constraint as well?

Reply:
Yes, a primary key constraint is basically a type of unique constraint, so you wouldn't need both.

Reply:
The frontend is currently set up to use a single ID for the entity, as opposed to knowing both species and project ID. What would you say are the benefits of doing it that way?

Reply:
Having a separate ID is not a problem, and may be the right approach if we want to be able to individually address one of these rows with a single ID value independently of its project and species IDs.

The benefits of a compound primary key are mostly about reducing redundancy: the rows are already uniquely addressable even without the synthetic ID. For very large tables the space savings can also be desirable, and for very write-heavy tables it can be worthwhile to have one less index that has to be updated, but those are obviously not relevant factors here.

Reply:
Thanks for the info, we will be interacting with this directly a lot, so I will leave it as is.